extension/main.ts

```diff
@@ -25,6 +26,60 @@ export function activate(context: ExtensionContext) {
     extension.onActivate();
 }
 
+class SetNextStatementHelpers
```

Comment:
Seems like all of that would be unnecessary, because the adapter will only ever generate one target?